src/collection-facets.ts

```diff
@@ -352,11 +352,17 @@ export class CollectionFacets extends LitElement {
               facetGroup.key !== 'collection'
                 ? html`${bucket.displayText ?? bucket.key}`
                 : html`
-                    <async-collection-name
-                      .collectionNameCache=${this.collectionNameCache}
-                      .identifier=${bucket.key}
-                      placeholder="-"
-                    ></async-collection-name>
+                    <a
+                      href="/details/${bucket.key}"
```

Comment:
we can hold here for now... when @nsharma123 gets his more facets work in, we can get `baseHost` to prepend